b2b_personalization/infra/_env/event_bridge.hcl

```
@@ -0,0 +1,45 @@
+locals {
+  common_vars = yamldecode(file(find_in_parent_folders("common_vars.yaml")))
+  env_vars    = read_terragrunt_config(find_in_parent_folders("env.hcl"))
+}
+
+terraform {
+  source = "github.com/terraform-aws-modules/terraform-aws-eventbridge//.?ref=v2.1.0"
+}
+
+dependency "slack" {
+  config_path = "../../slack_notifications"
+
+  mock_outputs = {
+    sns_slack_topic_arn = "arn:aws:sns:eu-central-1:123123123:slack-alarm-notification"
+  }
+  mock_outputs_allowed_terraform_commands = ["validate", "providers"]
+}
+
+inputs = {
+
+  create_bus = false
+
```

since we are using the default bus, the name of the role will be just `default`. 
```suggestion
  role_name         = "ecs-eventbridge"
```

---

can you please check if it still work with `create_role             = false `